client/src/pages/demo/components/index.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/no-unescaped-entities */
+import SideBar from '@/src/shared/components/SideBar/SideBar';
+
+import { Fragment } from 'react';
+
+interface componentsProps {
+  onSearch: string
+}
+
+const Components: React.FunctionComponent<componentsProps> = () => {
+  return (
+    <Fragment>
+      <div className='p-4 border-b border-black'>
```

Comment:
please add id in parent div for easy navigation, and also update the PR description for this.

<div className='p-4 border-b border-black' id="Sidebar">

apply this to all PRs